src/test/java/edu/cornell/kfs/concur/batch/fixture/ConcurStandardAccountingExtractFileFixture.java

```diff
@@ -37,6 +37,7 @@ private static ConcurStandardAccountingExtractDetailLine buildConcurStandardAcco
         ConcurStandardAccountingExtractDetailLine line = new ConcurStandardAccountingExtractDetailLine();
         line.setJounalDebitCredit(debitCredit);
         line.setJournalAmount(amount);
+        line.setEmployeeGroupId("CORNELL");
```

Comment:
Shouldn't this be using the parameter value lookup for CONCUR_CUSTOMER_PROFILE_GROUP_ID?